systems/plants/dev/RigidBodySpatialForce.m

```diff
@@ -0,0 +1,53 @@
+classdef RigidBodySpatialForce < RigidBodyForceElement
+  
```

Comment:
Actually, what would make that solution significantly better is that it doesn't access kinsol.J directly, which would not be available if the kinsol is obtained from mex.